rubocop.yml

```diff
@@ -792,6 +794,11 @@ Style/SelectByRegexp:
 Style/SingleArgumentDig:
   Enabled: false
 
+<% if rubocop_version >= Gem::Version.new('1.57') %>
+Style/SingleLineDoEndBlock:
+  Enabled: true
+<% end %>
```

Comment:
@rafaelfranca Trying something new here.

Typically, when a new cop is released, we enable or disable it, but that requires bumping the minimum RuboCop version (as older versions will complain they don't know about that cop).

As per our discussion about plugins, we'll have to take this approach to selectively enable plugin cops, as we won't be able to rely on Bundler/Rubygems to enforce any version requirement (without adding a dependency from `rubocop-shopify` to the various plugins, which we don't want to do).

Therefore, I'm trying out this approach here first.

Note that `rubocop-shopify.gemspec` has not changed, and CI is passing across all versions tested.